dcos/marathon.py

```diff
@@ -54,7 +54,11 @@ def _to_error(response):
     if isinstance(response, Error):
         return DefaultError(_default_marathon_error(response.error()))
 
-    message = response.json().get('message')
+    try:
+        message = response.json().get('message')
+    except ValueError as e:
```

Comment:
That isn't true for the user error that you're addressing here.  There is no 'message' field.  That's the source of the bug.  Can you point me to the marathon docs for this 'message' field?